src/services/OrderService.ts

```diff
@@ -570,12 +577,29 @@ const fetchGiftCardActivationDetails = async ({ isDetailsPage, currentOrders }:
 
   return isDetailsPage ? orders[0] : orders
 }
+const getBillingDetails = async (payload: any): Promise<any> => {
+  return api({
+    url: `/poorti/orders/${payload.orderId}/billing`,
+    method: "GET",
+    data: payload
```

Comment:
I think there is not need to pass payload separately, as the endpoint only takes orderId which is passed as a part of the endpoint.

If there is need to pass payload pass it as `params: payload`